backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/invoicecandidate/C_Invoice_Candidate_StepDef.java

```diff
@@ -162,6 +162,8 @@ public void validate_commission_settlement_invoice_cand(@NonNull final DataTable
 			final String invoiceCandIdentifier = DataTableUtil.extractStringForColumnName(row, COLUMNNAME_C_Invoice_Candidate_ID + "." + TABLECOLUMN_IDENTIFIER);
 			final I_C_Invoice_Candidate invoiceCandidate = invoiceCandTable.get(invoiceCandIdentifier);
 			assertThat(invoiceCandidate).isNotNull();
+			
+			InterfaceWrapperHelper.refresh(invoiceCandidate);
```

Comment:
this is not from me. Is seems to be an attempt to fix the build tests that caused the cucumber tests to give unstable build.